web/styles/message_row.css

```diff
@@ -143,7 +143,7 @@
         grid-area: time;
     }
 
-    .message_content {
+    .message_content:not(.status-message) {
         grid-area: message;
         padding: var(--message-box-markdown-aligned-vertical-space) 0 0;
         color: var(--color-text-message-default);
```

Comment:
This isn't related to this PR, but the `min-height: 17px` just below this seems suspicious in not having been moved to `em`.